docs/how-to/deploy_cups_sdcore.md

```diff
@@ -0,0 +1,217 @@
+# Deploy SD-Core with CUPS
```

Comment:
What do you think about spelling out what CUPS means? Everytime I see this, I think about the print server...

Reply:
If we really want to use this accronym, we should introduce it. Ex.

```
This guide covers how to install a multi-node sdcore with Control Plane and User Plane Separation (CUPS)
```